index.js

```diff
@@ -758,7 +758,7 @@ module.exports = function (log, indexesPath) {
   function getRecord(seq, cb) {
     const offset = indexes['seq'].tarr[seq]
     log.get(offset, (err, value) => {
-      if (err && err.code === 'flumelog:deleted') cb()
+      if (err && err.code === 'flumelog:deleted') cb(null, { seq, offset })
```

Comment:
This is the only thing needed for the fix, the other changes are style changes.

Reply:
nice find